### ISSUE TITLE
[Docs] Are the two convolutional neural network settings used in the ASTER model the same?

### ISSUE THREAD

**Issue by OuXiao514** [open-mmlab/mmocr#1921]
### Branch

master branch  https://mmocr.readthedocs.io/en/latest/

### 📚 The doc issue

Are the two convolutional neural network settings used in the ASTER model the same?
backbone=dict(
        type='ResNet',
        in_channels=3,
        stem_channels=[32],
        block_cfgs=dict(type='BasicBlock', use_conv1x1='True'),
        arch_layers=[3, 4, 6, 6, 3],
        arch_channels=[32, 64, 128, 256, 512],
        strides=[(2, 2), (2, 2), (2, 1), (2, 1), (2, 1)],
        init_cfg=[
            dict(type='Kaiming', layer='Conv2d'),
            dict(type='Constant', val=1, layer='BatchNorm2d'),
        ])

### Suggest a potential alternative/fix

Are the two convolutional neural network settings used in the ASTER model the same?

**Comment by Mountchicken** [comment]
Hi @OuXiao514 
The configuration of ResNet is the same as in the original paper.
